src/components/swapv2/SwapSettingsPanel/LiquiditySourcesSetting.tsx

```diff
@@ -69,7 +63,9 @@ const LiquiditySourcesSetting: React.FC<Props> = ({ onClick }) => {
       </Group>
 
       <Group>
-        <NumberOfSources>{numberOfDEXes}</NumberOfSources>
+        <NumberOfSources>
+          {selectedDexes.length || numberOfDEXes} out of {numberOfDEXes}
```

Comment:
If I deselect all DEXes, this will fall back to `numberOfDEXes`. Is it intentional?

Reply:
yes, it's intentional. If it show `0 out of 21`, it will weird because we treat zero as all dexes